### PR TITLE
Compound namespaces with a depth of *three* or more MUST not be used.

### PR DESCRIPTION
**Pull request by judgej**
It says:

> Compound namespaces with a depth of two or more MUST not be used.

Then gives some examples of compound namespaces with a depth of two, stating that they are valid. If they really are valid, then the depth that cannot be used is *three* or more. This is a one-indexed reality and not zero-indexed ;-)

### CODE REVIEW

**Comment by judgej** [comment]
Hey, I like the way github now looks after the fork when creating typo-PRs through the GUI. It used to force you to create the fork in your own account, which was a pain if you wanted multiple forks, and then it inevitably was left hanging around for far too long. I'm rambling...

**Comment by samdark** [comment]
You're correct about the fact that text contradicts examples.

File: proposed/extended-coding-style-guide.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ class FooBar
 
 ~~~
 
-Compound namespaces with a depth of two or more MUST not be used. Therefore the
+Compound namespaces with a depth of three or more MUST not be used. Therefore the
```
Comment (michaelcullum) [inline proposed/extended-coding-style-guide.md]: Can we rephrase this to `with a depth of more than two`

**Comment by judgej** [comment]
@michaelcullum yes, that is a much clearer way of putting it.

**Comment by michaelcullum** [comment]
👍
